litmus-portal/frontend/src/models/user.ts

```diff
@@ -23,5 +41,7 @@ export type UserAction =
   | UserActionType<typeof UserActions.LOGOUT_USER, string>
   | UserActionType<
       typeof UserActions.UPDATE_USER_DETAILS,
-      { name: string; email: string; projectName: string }
+      {
+        selectedProjectID: string;
```

Comment:
The payload needs to be put in an interface if it is not a primitive type like string or number